elections_process/process.md

```diff
@@ -0,0 +1,63 @@
+# Jupyter Distinguished Contributors Elections process
+
+## 1] Call for elections shepherds
+
+The shepherd(s) of 20XX-1 JDC election should send an email to the [JDC mailing list](jupyter-distinguished-contributors@googlegroups.com) in early December 20XX, asking for volunteers to shepherd the 20XX election. Any JDC can participate in administering the JDC election, and the 
+
+## 2] Nominations
+
+The nominations for 20XX JDC should start in early January 20XX+1. The shepherd(s) should send an email to the [JDC mailing list](jupyter-distinguished-contributors@googlegroups.com) with the following elements:
+
+- Number of available awards (currently 10 per year)
+- A link to the form used for nominating someone. A Google Form template has been shared with the JDC Google groups, you can duplicate it (please do not use it directly so it can be reused for the next elections); you can also create a new form from [this document](https://github.com/jupyter/distinguished-contributors/blob/main/nomination-forms/2020.md)
+- A reminder of the criteria for [Distinguished Contributor awards](https://jupyter.org/governance/distinguished_contributors.html#criteria-for-nominating-new-distinguished-contributors). The 2-year activity requirement should be as of December 31, 20XX.
+- A deadline for the nomination phase; this phase should not exceed 2 or 3 weeks.
+
+The elections shepherd(s) should not hesitate to send some reminders during the nomination phase.
+
+## 3] Vote
+
+Once the nomination phase is over, the vote can start. There are two cases:
+
+- if the number of nominees does not exceed the number of available awards, a simple form with "Yes / No / Abstain" options for each nominee can be used.
```

Comment:
Probably worth being explicit- is `Abstain` completely ignored so the only requirement is there are more `Yes` than `No` votes?